Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.10 as builder
+
+WORKDIR ${GOPATH}/src/github.com/dispatchframework/dispatch-events-aws
+
+COPY ["driver.go", "Gopkg.lock", "Gopkg.toml", "./"]
+
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN dep ensure
+
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o dispatch-events-aws
+RUN cp ./dispatch-events-aws /dispatch-events-aws
```

Comment:
You can remove the line with `cp` and change the `go build` output to `/dispatch-events-aws`